client/src/components/Form/Form.vue

```diff
@@ -1,31 +1,68 @@
 <template>
```

Comment:
It's not specifically for tools but wraps all forms within Galaxy e.g. when you create a new workflow, or a new page or new user, or set user preferences and so on. Also that is how this element was historically named in Backbone so I wanted to keep the name consistent such that it can be easily found and associated after the legacy Backbone code has been removed but I am open to suggestions.